tests/integration/test_charm.py

```diff
@@ -244,18 +248,18 @@ async def get_postgres_password(ops_test: OpsTest):
     return result.results["postgres-password"]
 
 
-async def pull_content_from_unit_file(unit, path: str) -> str:
-    """Pull the content of a file from one unit.
+async def get_unit_address(ops_test: OpsTest, unit_name: str):
```

Comment:
put all non tests in a file `tests/integration/helpers.py` (can be done in subsequent PR but don't forget about it)

Reply:
Of course. The new functions were already created in the helpers and the remaining will be moved to there.

Reply:
bump